src/config.js

```diff
@@ -115,9 +123,10 @@ class Config {
    * @param {number} creditLimit  Number of operations to permit every `resetSeconds`
    * @param {number} resetSeconds Credit renewal interval.
    * @param {string} actorField   Name of the actor field (optional).
+   * @param {string} label        Optional name for this rule.
    * @param {string} comment      Optional diagnostic name for this rule.
    */
-  addRule(operation, creditLimit, resetSeconds, actorField, comment) {
+  addRule(operation, creditLimit, resetSeconds, actorField, label, comment) {
```

Comment:
since label is an optional argument, why position it second to last? 
there's a bunch of places in `tests/config-test.js` that will need to be updated to reflect that positional change.

alternatively, since the cardinality of this function is getting big, what are your thoughts on having it accept an arguments object instead of positional arguments?

Reply:
> since label is an optional argument, why position it second to last?

Since both `label` and `comment` are optional, am guessing I somewhat arbitrarily decided labels will be more common than comments.

> alternatively, since the cardinality of this function is getting big, what are your thoughts on having it accept an arguments object instead of positional arguments?

I like it, that'd definitely make it a bunch more readable! May sheepishly defer this suggestion as am running into limited cycles to iterate further. OK with you to land as is?